mage_ai/frontend/components/datasets/FeatureProfiles/index.tsx

```diff
@@ -155,9 +155,9 @@ function FeatureProfile({
             >
               <Text
                 bold
+                color={themeContext.chart.feature}
```

Comment:
Can you just use the `default` prop to use this font color? And in the case of the data cleaning tool, add a conditional on the `secondary` prop, so it will make the font color purple for the link but not in the data prep tool where there is no link.